libs/openFrameworksCompiled/project/android/build.gradle

```diff
@@ -14,10 +14,11 @@ model {
 
         defaultConfig.with {
             minSdkVersion.apiLevel 19
-            targetSdkVersion.apiLevel 19
+            targetSdkVersion.apiLevel 25
         }
 
         ndk {
+            platformVersion = "19"
```

Comment:
Found out this is the trick to keep NDK compiling against 19, while bumping java to 25.

Reply:
Found solution here: https://github.com/android-ndk/ndk/issues/126#issuecomment-334672371

Reply:
Found solution here: https://github.com/android-ndk/ndk/issues/126#issuecomment-334672371